sdk/formrecognizer/ai-form-recognizer/src/models.ts

```diff
@@ -345,6 +372,14 @@ export interface RecognizedForm {
    * Document type.
    */
   formType: string;
+  /**
+   * Confidence in the correctness of the form type.
```

Comment:
if this is the docstring for the user, consider including more information about how it will not have a value for unlabeled models. i.e. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedForm.cs#L84